src/test/java/no/nav/veilarbvedtaksstotte/service/BeslutterServiceTest.java

```diff
@@ -65,7 +73,7 @@ public void startBeslutterProsess__skal_starte_beslutter_prosess_hvis_ikke_start
 
         beslutterService.startBeslutterProsess(SOME_ID);
 
-        verify(vedtaksstotteRepository, times(1)).setBeslutterProsessStatus(anyLong(), eq(KLAR_TIL_BESLUTTER));
+        verify(vedtaksstotteRepository, atLeastOnce()).setBeslutterProsessStatus(anyLong(), eq(KLAR_TIL_BESLUTTER));
```

Comment:
Bruk heller times(1). `atLeastOnce()` vil si at det er greit hvis funksjonen kalles 1 eller flere ganger, men vi ønsker kun at den skal kalles 1 gang. Dette gjelder også for de andre testene i BeslutterService.